server/oc_cmds.go

```diff
@@ -5,15 +5,17 @@ package server
 import (
 	"fmt"
 
+	sdk "github.com/line/lbm-sdk/types"
+	cfg "github.com/line/ostracon/config"
+	osjson "github.com/line/ostracon/libs/json"
+	ostos "github.com/line/ostracon/libs/os"
+	"github.com/line/ostracon/node"
 	"github.com/line/ostracon/p2p"
 	pvm "github.com/line/ostracon/privval"
+	"github.com/line/ostracon/types"
 	ostversion "github.com/line/ostracon/version"
 	"github.com/spf13/cobra"
 	yaml "gopkg.in/yaml.v2"
-
-	"github.com/line/lbm-sdk/client"
-	cryptocodec "github.com/line/lbm-sdk/crypto/codec"
-	sdk "github.com/line/lbm-sdk/types"
 )
```

Comment:
Please follow the rules for separating imports by group. 
1. go library (ex: os, rand, fmt)
2. other github path (ex: github.com/spf13/cobra)
3. line github path (ex: github.com/line/ostracon/node)
4. current path (ex: github.com/line/lbm-sdk/types)

Reply:
I fixed as indicated.